pywps/configuration.py

```diff
@@ -135,20 +132,25 @@ def load_configuration(cfgfiles=None):
     CONFIG.add_section('grass')
     CONFIG.set('grass', 'gisbase', '')
 
+    CONFIG.add_section("watchdog")
+    CONFIG.set("watchdog", "pause", "30")
```

Comment:
In the documentation, this section is called `daemon`.
Also, you could add the `pidfile` default configuration here?

Reply:
updated to `jobqueue` in docs and code.